build/spas.ts

```diff
@@ -238,8 +240,9 @@ export async function buildSPAs(options) {
       fs.writeFileSync(filePathContext, JSON.stringify(context));
     }
   }
+
   await buildStaticPages(
-    path.join(dirname, "../copy/plus"),
+    path.join(dirname, "../copy/plus/"),
```

Comment:
Probably not necessary anymore.

```suggestion
    path.join(dirname, "../copy/plus"),
```

Reply:
No, checked again, omitting is better, to avoid duplicate slashes in the `const url` near line 213.